.github/workflows/integration.yaml

```diff
@@ -370,3 +370,166 @@ jobs:
       - name: Run tests
         run: |
           cd $(go env GOROOT)/src/os; wazero run -mount=/:/ ~/bin/os.wasm -test.v
+
+  build_go_tests:
+    name: Build Go test binaries
+    runs-on: ubuntu-22.04
+    env:
+      BINARY_OUT: ~/go-test-binaries
+
+    steps:
+      # - name: Clear out cache 
```

Comment:
wonder if we shouldn't add this to the clear_cache workflow and add workflow_dispatch so that we can trigger manually instead of having commented out code?

Reply:
I can try. I'm new to GH workflow so it's a nice learning opportunity